### PR TITLE
Configurable logging, runtime level change

### PR DESCRIPTION
**Pull request by andrewmed**
Configurable logging, runtime level change

### CODE REVIEW

File: logger/logger.go [hunk 1]
```diff
@@ -2,30 +2,54 @@ package logger
...
-		zapcore.AddSync(os.Stdout),
-		zap.NewAtomicLevelAt(zap.InfoLevel),
-	),
-).Sugar().Named("fd")
+var Instance *zap.SugaredLogger
+var Level zap.AtomicLevel
+
+func init() {
+	level := zap.InfoLevel
+	switch strings.ToLower(os.Getenv("LOG_LEVEL")) {
```
Comment (ssnd) [inline logger/logger.go:16]: I still think the initial debug level should be passed through command line options, not environment variables. Would love to hear @vano144 @ansakharov 's opinions on this.   
If we decide however to stick to your option, we should document it well. Now this just looks like some undocumented magic only file.d developers are aware of.

File: logger/logger.go [hunk 2]
```diff
@@ -2,30 +2,54 @@ package logger
...
+	case "fatal":
+		level = zap.FatalLevel
+	}
+
+	Level = zap.NewAtomicLevelAt(level)
+
+	Instance = zap.New(
+		zapcore.NewCore(
+			zapcore.NewConsoleEncoder(zapcore.EncoderConfig{
+				// TimeKey:        "ts",
```
Comment (ssnd) [inline logger/logger.go:34]: ?

File: fd/file.d.go [hunk 3]
```diff
@@ -265,6 +265,7 @@ func (f *FileD) startHTTP() {
 	mux.HandleFunc("/ready", f.serveLiveReady)
 	mux.HandleFunc("/freeosmem", f.serveFreeOsMem)
 	mux.Handle("/metrics", promhttp.Handler())
+	mux.Handle("/log/level", logger.Level)
```
[the remark this answers is not in the document]
Comment (andrewmed) [reply fd/file.d.go:268]: Done

File: logger/logger.go (hunk 1 above)
Comment (andrewmed) [reply logger/logger.go:16]: Done
Added to documentation

**Pull request closed without merging**